### PR TITLE
New package: Kitware.ParaView.MPI version 5.11.2

### LINKED ISSUE #122435: [Package Request]: Microsoft.MPI
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.microsoft.com/en-us/download/details.aspx?id=57467
Publisher: Microsoft
Package Name: MPI
Description: MS-MPI enables you to develop and run MPI applications without having to set up an HPC Pack cluster.
Package Version: 10.0.12498.5
Installer URL:
EXE: https://download.microsoft.com/download/A/E/0/AE002626-9D9D-448D-8197-1EA510E297CE/msmpisetup.exe

Needed for: 
- https://github.com/microsoft/winget-pkgs/pull/122402

### PR DESCRIPTION
**Pull request by Dragon1573**
## What's changed

This is an MPI accelerated version of `Kitware.ParaView`.

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122402)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @Dragon1573,

The package installed normally, but failed to launch in a Windows 10 VM:
image

This appears to be a dependency on [Microsoft MPI](https://www.microsoft.com/en-us/download/details.aspx?id=57467), which doesn't appear to have a public manifest yet.

File: manifests/k/Kitware/ParaView/MPI/5.11.2/Kitware.ParaView.MPI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Kitware.ParaView.MPI
+PackageVersion: 5.11.2
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+ProductCode: '{76D57FB1-0CD5-40A2-9296-16B85344BCAA}'
+AppsAndFeaturesEntries:
+- UpgradeCode: '{E06445A7-B257-4FCE-9241-2A189AD26B5A}'
```
Comment (stephengillie) [inline manifests/k/Kitware/ParaView/MPI/5.11.2/Kitware.ParaView.MPI.installer.yaml:11]:
```suggestion
- UpgradeCode: '{E06445A7-B257-4FCE-9241-2A189AD26B5A}'
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.MPI
```
Once the Issue is resolved, this change should resolve the application error. (Before the issue is resolved, this should cause a slightly different error due to the missing dependency.)

**Comment by stephengillie** [comment]
Unfortunately, with MS-MPI preinstalled, the application crashes on launch. 

Manual Validation ended with:
> Faulting application name: paraview.exe, version: 0.0.0.0, time stamp: 0x6511a217  
> Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000 Exception code: 0xc0000005 
Fault offset: 0x0000000000000000  
> Faulting process id: 0x9e0  
> Faulting application start time: 0x01d9fc9e5a1376a4  
> Faulting application path: C:\Program Files\ParaView 5.11.2\bin\paraview.exe  
> Faulting module path: unknown Report Id: b446f8ae-ac45-488d-a595-920bb8915780  
> Faulting package full name:   
> Faulting package-relative application ID:

**Comment by Dragon1573** [comment]
> Unfortunately, with MS-MPI preinstalled, the application crashes on launch.

Does it mean `Kitware.ParaView.MPI` still have something unusual and cannot be accepted by `winget-pkgs`?

If this problem is still there and could not be resolved, I think we can close this pull request. `Kitware.ParaView` (without MPI acceleration) is already available in `winget-pkgs`.

**Comment by stephengillie** [comment]
It turns out that MS-MPI already exists, as `Microsoft.msmpi`. Sorry for not checking more thoroughly.

**Comment by stephengillie** [comment]
I've updated the suggestion above. If you will commit it then the pipeline can retry.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**